File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/models/TrompetistaTest.java
package simulacionBandaMusica.models;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.factories.TrompetistaFactory;

import static org.junit.jupiter.api.Assertions.assertEquals;

class TrompetistaTest {

    @Test
    void respirarTrompetista() {
        Trompetista trompetistaTest = TrompetistaFactory.create();
        String correctoString = "Estoy respirando como trompetista con una capacidad pulmonar de " + trompetistaTest.capacidadPulomar;
        assertEquals(correctoString, trompetistaTest.respirarTrompetista());
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/models/TrompetistaTest.kt
package simulacionBandaMusica.models

import org.junit.jupiter.api.Test
import simulacionBandaMusica.factories.TrompetistaFactory
import kotlin.test.assertEquals

internal class TrompetistaTest {
    @Test
    fun respirarTrompetista() {
        val trompetistaTest: Musico.Trompetista = TrompetistaFactory.create()
        val correctoString =
            "Estoy respirando como trompetista con una capacidad pulmonar de " + trompetistaTest.capacidadPulomar
        assertEquals(correctoString, trompetistaTest.respirarTrompetista())
    }
}